scripts/install.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+### usage
+# mac/linux: /bin/bash -c "$(curl -fsSL https://gist.githubusercontent.com/solarw/b1fbb518c63b62bb056f502fa951ed1a/raw/install.sh)"
+
+function bad_os_type() {
+	echo "OS $OSTYPE is not supported!"
+	exit 1
+}
+
+function check_linux() {
+	# check any deb distribution!
+	is_ubuntu=`cat /etc/issue|grep -i Ubuntu`
+	if  [[ -z $is_ubuntu ]];
+	then
+		echo "Only ubuntu, macos are supported at the moment with this script. please use install.ps1 for windows 10"
+		exit 1
+	fi
+	install_on_ubuntu
+}
+
+
+function is_python3(){
+	return `which python3`
+}
+
+function is_python_version_ok() {
+	if which python3 2>&1 >/dev/null;
+	then
+		version=`python3 -V 2>/dev/null`
+		if [[ -z `echo $version|grep -E 'Python 3\.[(678]\.[0-9]+'` ]];
+		then
+			echo "Python3 version: ${version} is not supported. Supported versions are 3.6, 3.7, 3.8."
+			return 1
+		fi
+		return 0
+	else
+		echo "Python is not installed"
+		return 1
+	fi
+}
+
+
+function install_aea (){
+	echo "Install AEA"
+	output=$(pip3 install aea[all])
+	if [[  $? -ne 0 ]];
+	then
+		echo "$output"
+		echo 'Failed to install aea'
+		exit 1
+	fi
+	
+	output=`aea --help 2>&1`
+	if [[  $? -ne 0 ]];
+	then
+		echo "$output"
+		echo 'Test run of aea failed!'
+		exit 1
+	fi
+	echo "AEA successfully installed!"
```

Comment:
![image](https://user-images.githubusercontent.com/9467617/96625004-de4ca680-130d-11eb-896f-55abfdda9e09.png)

To reproduce:
```
docker run -it -v$(pwd):/home/build ubuntu:20.04
# now in image
apt-get update
apt-get install -y sudo
useradd newuser
passwd newuser  # type any password
usermod -aG sudo newuser
su newuser
bash
cd /home/build
sudo ./script/install.sh
```


we could extend PATH somewhere before `pip install` or right after

Reply:
Interestingly, if you run `sudo ./script/install.sh`, it will run `pip` with `sudo`, hence installin `aea` globally, instead of doing an user installation (hence installing it in `~/.local`).